src/indexPage/endpoint.js

```diff
@@ -64,6 +64,16 @@ function addEndpoint(e) {
     setStorage('config.server_url', serverUrl);
     setStorage('config.app_id', appId);
 
+    // eslint-disable-next-line no-useless-escape
+    const UrlReg = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/;
```

Comment:
- In the current Binary Bot, we use camelCase notation starting the first word with a small letter.
- You can remove the `eslint` exception by removing the useless escapes in the regex pattern, i.e. you don't have to escape special characters in a range.